include/ledger_assert.h

```diff
@@ -2,6 +2,10 @@
 
 #include <stdbool.h>
 
+#ifdef HAVE_PRINTF
+#include <os.h>
```

Comment:
```suggestion
#include "os.h"
```
Cause it's not a toolchain header but a SDK one?
Though we don't follow any strict rules about this.